### DISCUSSION TITLE
[Question] Closest to MAUI

### DISCUSSION THREAD

**Discussion by libra34567** [dotnet/maui#3010] [Q&A]
Coming from  Unity3D background. I'm a total beginner comes to application development. 

I love the idea of MAUI, but at the time I'm writing this issue, its just seems too early/unstable and too little tutorials/resources I could find for MAUI. 

I wonder if there's any alternative that has more tutorial/resource/userBase. That would also have the minimum difference and easy transfer to MAUI? I want to use that until MAUI is more popular and transfer the project to MAUI. 

Looking forward to any suggestions!

**Comment by JohnHDev** [comment]
It sounds like Xamarin.Forms is what you are after.
